flink-kubernetes-operator/src/test/java/org/apache/flink/kubernetes/operator/reconciler/deployment/ApplicationReconcilerTest.java

```diff
@@ -654,28 +653,6 @@ private void testSnapshotLegacy(FlinkDeployment deployment, SnapshotType snapsho
         assertTrue(isSnapshotInProgress.test(getJobStatus(snDeployment)));
         assertEquals(SnapshotStatus.PENDING, getLastSnapshotStatus(snDeployment, snapshotType));
         snDeployment.getSpec().getFlinkConfiguration().put(triggerSnapshotExpression.key(), "0");
-
```

Comment:
I have deleted this test case as I could not find a good way to update the recorded last periodic timestamp without modifying access to the `SnapshotTriggerTimestampStore` fields. 

We already test the CRON ability by testing methods `SnapshotUtils#shouldTriggerAutomaticSnapshot` and `SnapshotUtils#interpretAsCron`.